src/pyuvdata/uvbeam/uvbeam.py

```diff
@@ -2632,6 +2615,9 @@ def __add__(
         run_check_acceptability : bool
             Option to check acceptable range of the values of
             required parameters after combining objects.
+        warn_spacing : bool
+            Option to raise warnings about spacing that would prevent writing to
+            calfits file-format. Default is True.
```

Comment:
calfits -> beamfits

Reply:
Fixed in e42deb6a26a1f3e58ccfe3436a40ca41022ccea0.